### DISCUSSION TITLE
[Question][Hw4] A question about small test cases in HW4

### DISCUSSION THREAD

**Discussion by kimjungwow** [prosyslab-classroom/cs348-information-security#594] [Q&A]
Hello! 😁

I have a question about small test cases in HW4.

This [document](https://docs.google.com/document/d/1zIM83xkeyy8lBDJa1C6QVWM2rqUFxoyX9FIHqbQTSCg/edit) said **"We will use the same seed inputs in the repository for evaluation."**.
Doesn't it mean `test/string_seed/input.txt` would be used for evaluation?
It seems `run_fuzzer.py` uses `test/test_seed/seed.txt` and `test/test_seed/seed2.txt` (which are newly pushed) for evaluation.
Please let me know if there is anything I misunderstood.

Thank you!

**Comment by Junseon00** [comment]
Yes, you're right. Sorry for the confusion, we'll re-grade for the small testcases. Thank you!

**Comment by kimjungwow** [comment]
Thank you!
